src/py/flwr/cli/run/run.py

```diff
@@ -15,46 +15,29 @@
 """Flower command line interface `run` command."""
 
 import sys
-from enum import Enum
 from logging import DEBUG
 from pathlib import Path
-from typing import Dict, Optional
+from typing import Optional
 
 import typer
 from typing_extensions import Annotated
 
-from flwr.cli import config_utils
 from flwr.cli.build import build
+from flwr.cli.config_utils import load_and_validate
 from flwr.common.config import parse_config_args
-from flwr.common.constant import SUPEREXEC_DEFAULT_ADDRESS
 from flwr.common.grpc import GRPC_MAX_MESSAGE_LENGTH, create_channel
 from flwr.common.logger import log
 from flwr.proto.exec_pb2 import StartRunRequest  # pylint: disable=E0611
 from flwr.proto.exec_pb2_grpc import ExecStub
 from flwr.simulation.run_simulation import _run_simulation
 
 
-class Engine(str, Enum):
-    """Enum defining the engine to run on."""
-
-    SIMULATION = "simulation"
-
-
 # pylint: disable-next=too-many-locals
 def run(
-    engine: Annotated[
-        Optional[Engine],
-        typer.Option(
-            case_sensitive=False,
-            help="The engine to run FL with (currently only simulation is supported).",
-        ),
+    address: Annotated[
+        Optional[str],
+        typer.Option(help="The address of the SuperExec to run on"),
```

Comment:
This should not be a CLI arg, but it would instead be read from `[flower.federations]`. However, we will soon need two new positional CLI args: `app_path` (usually provided as `.` and `federation_name`).